scripts/bee.sh

```diff
@@ -229,12 +249,20 @@ for i in $(seq 1 1 "$WORKERS"); do
 done
 
 echo "Check whether the queen node has been connected to every worker..."
-while : ; do
+ELAPSED_TIME=0
+WAITING_TIME=2
+# Wait 2 mins for queen start
+TIMEOUT=$((2*30*WAITING_TIME))
```

Comment:
Same comments as above. IMHO `TIMEOUT` should be constant and `WAITING_TIME` renamed to `RETRY_PERIOD`

Reply:
I think the `RETRY_PERIOD` is more ambiguous than `WAITING_TIME`

Reply:
OK. Then my suggestion would be to write a line commend after this variable definition explaining what it does. So here it would be

```
WAITING_TIME=2 # Time in seconds between checks if the worker is connected to queen bee
```